Documents/API.md

```diff
@@ -232,58 +232,71 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
       "syntactical": {
         "edits": [
           {
-            "edit": "S025",
-            "description": "Description of S025",
-            "ts": true,
-            "lars": [
+            "edit": "S020",
+            "description": "Agency code must = 1, 2, 3, 5, 7, 9. The agency that submits the data must be the same as the reported agency code.",
+            "rows": [
               {
-                "lar": {"loanId": "s1"}
+                "row": { "rowId": "Transmittal Sheet" }
               },
               {
-                "lar": {"loanId": "s2"}
+                "row": { "rowId": "8299422144" }
               },
               {
-                "lar": {"loanId": "s3"}
+                "row": { "rowId": "2185751599" }
               }
             ]
           },
           {
             "edit": "S010",
-            "description": "Description of S010",
-            "ts": false,
-            "lars": [
+            "description": "The first record identifier in the file must = 1 (TS). The second and all subsequent record identifiers must = 2 (LAR).",
+            "rows": [
               {
-                "lar": {"loanId": "s4"}
-              },
+                "row": { "rowId": "2185751599" }
+              }
+            ]
+          }
+        ]
+      },
+      "validity": {
+        "edits": [
+          {
+            "edit": "V555",
+            "description": "If loan purpose = 1 or 3, then lien status must = 1, 2, or 4.",
+            "rows": [
               {
-                "lar": {"loanId": "s5"}
+                "row": { "rowId": "4977566612" }
               }
             ]
           }
         ]
       },
-      "validity": {},
-      "quality": {},
-      "macro": [
-        {
-            "edits": [
-                {
-                   "edit": "Q007",
-                   "description": "Description of Q007",
-                   "justifications": [
-                     {
-                       "value": "don't worry",
-                       "selected": false
-                     },
-                     {
-                       "value": "be happy",
-                       "selected": false
-                     }
-                   ]
-                }
+      "quality": {
+        "edits": []
```

Comment:
Good question. I checked on master, and it has the same output as what's shown here.:
```json
  "quality": {
    "edits": []
  }
```

So evidently the docs just hadn't caught up yet.